src/main/scala/org/zalando/kanadi/api/Subscriptions.scala

```diff
@@ -1061,12 +1060,12 @@ case class Subscriptions(baseUri: URI, authTokenProvider: Option[AuthTokenProvid
   }
 
   private final val killSwitches =
-    new ConcurrentHashMap[(SubscriptionId, StreamId), UniqueKillSwitch]().asScala
+    new ConcurrentHashMap[(SubscriptionId, StreamId), UniqueKillSwitch]()
```

Comment:
I decided to do this in order to remove the usage of `import scala.collection.JavaConverters._`. `scala.collection.JavaConverters` is fine to use in Scala 2.12, is deprecated in 2.13 and doesn't exist in Scala 3 and although there is https://github.com/scala/scala-collection-compat to solve this issue I don't think its worth it to add an entirely new dependency just for something thats this trivial